ui/packages/app/src/components/StatusTracker/StatusTracker.svelte

```diff
@@ -191,13 +189,21 @@
 
 <style lang="postcss">
 	.wrap {
-		@apply absolute inset-0 z-50 flex flex-col justify-center items-center bg-white dark:bg-gray-800 pointer-events-none transition-opacity max-h-screen;
+		@apply absolute inset-0 z-50 flex flex-col justify-center items-center dark:bg-gray-800 pointer-events-none transition-opacity max-h-screen;
 	}
 
-	:global(.dark) .wrap {
+	:global(.dark) .cover-bg {
 		@apply bg-gray-800;
 	}
 
+	.cover-bg {
+		@apply bg-white;
+	}
+
+	.generating {
```

Comment:
The generating animation looks really cool @aliabid94 !

![orange_animation_small](https://user-images.githubusercontent.com/41651716/188992956-4b0fc1e6-3c0b-4263-9402-553664303c97.gif)

Reply:
Agreed!